client/src/components/river/index.tsx

```diff
@@ -138,7 +189,7 @@ const FhirRiverView = (): React.ReactElement => {
         <p>
           Choose all the resources you want to process with the ETL. If you
           select none of them, all we be processed. Note that you need to select
-          a source before.
+          a source before
```

Comment:
Could you remove the `If you select none of them, all we be processed.`, please 🙏